pkg/cloud/libvirt/client/client.go

```diff
@@ -425,7 +435,7 @@ func (client *libvirtClient) CreateVolume(input CreateVolumeInput) error {
 		// create the volume
 		// Refresh the pool of the volume so that libvirt knows it is
 		// not longer in use.
-		waitForSuccess("error refreshing pool for volume", func() error {
+		err = waitForSuccess("error refreshing pool for volume", func() error {
```

Comment:
This seems like an unrelated fix (to existing code). Please put it in a separate patch.

Reply:
Thanks for pointing out:) Actually I did not fix this in this branch manually, instead, I just merged the master branch into this branch before squashing & pushing. Anyway, fixed.

Reply:
it got unfixed when squashing other patches. :(

Reply:
> it got unfixed when squashing other patches. :(

I did it on purpose. I just want to make sure this pull request do not tangle with any other changes/pull requests.